src/js/providers/data-normalizer.ts

```diff
@@ -0,0 +1,17 @@
+import { GenericObject } from 'types/generic.type';
+
+export type QualityLevel = {
+    label: string;
+    bitrate?: number;
+    width?: number;
+    height?: number;
+};
+
+export function qualityLevel(level: QualityLevel | GenericObject): QualityLevel {
```

Comment:
`QualityLevel & GenericObject` ensures that it at least following `QualityLevel` and allows more props, doing the or means a blank object could be passed in and pass

Reply:
That makes more sense